test/jdk/jdk/jfr/event/gc/heapsummary/TestHeapSummaryEventDefNewSerial.java

```diff
@@ -34,6 +34,17 @@
  * @library /test/lib /test/jdk
  * @run main/othervm -XX:+UseSerialGC jdk.jfr.event.gc.heapsummary.TestHeapSummaryEventDefNewSerial
  */
+
+/**
+ * @test
+ * @bug 8264008
+ * @key jfr
+ * @requires vm.hasJFR
+ * @requires vm.gc == "Serial" | vm.gc == null
+ * @library /test/lib /test/jdk
+ * @run main/othervm -XX:+IgnoreUnrecognizedVMOptions -XX:+UseSerialGC -XX:-UseCompressedClassPointers
+ *                   jdk.jfr.event.gc.heapsummary.TestHeapSummaryEventDefNewSerial
```

Comment:
I would exclude 32bit here since it would just re-run the same test as above. Does the 32bit VM even accept UseCompressedClassPointers?

Reply:
> I would exclude 32bit here since it would just re-run the same test as above. Does the 32bit VM even accept UseCompressedClassPointers?

32bit has been excluded.
I don't think the 32bit VM accepts UseCompressedClassPointers and that's why there is -XX:+IgnoreUnrecognizedVMOptions in my previous patch.
Thanks.